net-misc/trackma/trackma-0.7.3.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{3_4,3_5} )
+
+inherit eutils distutils-r1
+
+SRC_URI="https://github.com/z411/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
Please keep the variables in order with the templates (skel.ebuild, vim or likewise); otherwise, it's hard to read it.